grid/example/src/cells/RowOptions.js

```diff
@@ -18,7 +20,17 @@ const RowOptions = memo((props) => {
         setOpen(false);
     };
 
-    return (
+    const showEditOverLay = () => {
+        console.log("showEditOverLay");
+        setOpen(false);
+        setOverLayOpen(true)
+    }
+
+    const setOverLayClose = () => {
+        setOverLayOpen(false)
+    }
+
+    let htmlReturnValue = isOverLayOpen === false ?

```

Comment:
I think we can replace the statement, isOverLayOpen === false
with
!isOverLayOpen 

Please check if that works (for code optimization)

Reply:
Done. Code is working fine with changes